addons/crm/static/tests/tours/crm_forecast_tour.js

```diff
@@ -20,7 +20,7 @@ registry.category("web_tour.tours").add('crm_forecast', {
         content: 'Open Forecast menu',
         run: 'click',
     }, {
-        trigger: '.o_column_quick_create:contains(Add next month)',
+        trigger: '.o_column_quick_create',
```

Comment:
That change could be introduced during the forward port of your other PR https://github.com/odoo/odoo/pull/207891